geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/GatewayReceiverCommand.java

```diff
@@ -303,8 +300,7 @@ public void cmdExecute(final Message clientMessage, final ServerConnection serve
                   handleMessageRetry(region, clientEvent);
```

Comment:
>On line 293 the compiler warning about raw use of parameterized class can be fixed by using

I have purposely stayed away from VersionTag generics. I went down that rabbit hole in other places and it is a mess. That type is a mess when it comes to generics. It relates to several other classes that are also a mess. Often times the wildcard capture results in all sorts of compile issues that then have to be resolved with more generic type matches and then ultimately some unchecked casts. I have a todo on my plate to try and fix this class.

Reply:
>On line 273 we're doing this:

I intentionally left that one in, and a few others similar to it, because it just felt more readable to increment as it goes down the parts collection. I can go either way with it.